aiohttp/web_server.py

```diff
@@ -69,6 +72,7 @@ def pre_shutdown(self) -> None:
     async def shutdown(self, timeout: Optional[float] = None) -> None:
         coros = (conn.shutdown(timeout) for conn in self._connections)
         await asyncio.gather(*coros)
+        print("LENGTH", len(self._connections))
```

Comment:
I'd like to assert the number of connections at this point, but can't think of a good way to get this in the tests. Any ideas?

Reply:
Maybe patch `conn.shutdown` and see how many calls are made to it?

Reply:
I don't think that works. The number of shutdown() calls doesn't matter, it's the number of them which didn't result in the connections being removed from the set as part of the done callback.

Reply:
I think if I can figure out how to get a reference to the Server object, then I could patch the .clear() call to check how many are still in the set when this is called.